examples/with-typescript/components/ListItem.tsx

```diff
@@ -1,12 +1,16 @@
 import * as React from 'react'
+import Link from 'next/link';
+
 import IDataObject from '../interfaces'
 
 type Props = {
   data: IDataObject,
 }
 
 const ListItem: React.FunctionComponent<Props> = ({ data }) => (
-  <React.Fragment>{data.id}:{data.name}</React.Fragment>
+  <Link href={`/detail?id=${data.id}`} passHref>
```

Comment:
passHref is not required on Link tags, or is it when using typescript?

Reply:
It should be fine without `passHref`, I'm just used to putting that in for `Link`s. 😅